test/e2e/node/events.go

```diff
@@ -20,16 +20,14 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/onsi/ginkgo"
```

Comment:
This is not necessary to be moved in this PR.
The original sort order is common in e2e tests.
That means we need to keep like
```
...
	"k8s.io/kubernetes/test/e2e/framework"

	"github.com/onsi/ginkgo"
```
with a single blank line between framework and ginkgo.